.circleci/config.yml

```diff
@@ -56,101 +44,39 @@ jobs:
                 command: npx percy finalize --all
                 when: always
 
-
-    "standalone-test":
```

Comment:
These tests have been ported to test-server / Selenium

Reply:
`test-server` is awfully long now, nearly an hour. Can we parallelize it, shoot for <10min?

Reply:
@alexcjohnson I've tried but for some reason one of the test consistently fails with parallelism turned on. I've got no idea why. Can try and figure it out before we consider this PR ready.

Reply:
Related, there's one set of tests that is extremely long to run with Selenium (lots of DOM checks). The minimal run time with per-file slicing is probably not lower than 15-20minutes because of that test and its variations.

Out of scope for this PR: it should be possible to rewrite that test to run the checks directly from the browser (Karma) instead and improve its speed by at least an order of magnitude.

Reply:
@alexcjohnson Figured out what I did wrong. Also, broke down the large test with lots of permutations into many smaller tests in their own files. The last run took ~28 minutes but I expect the CI time splitting algorithm to be able to optimize this down to below 20 minutes soon enough.

Reply:
> Figured out what I did wrong

Excellent, which change fixed this?

> I expect the CI time splitting algorithm to be able to optimize this down to below 20 minutes soon enough.

OK, that's reasonable for now, thanks!

Reply:
> which change fixed this?

When I updated the CI yaml to run tests in parallel, I copied it over from the DCC CI yaml which runs pytest with the `--headless` flag. For some reason `tbmu001`, specifically, doesn't pass when it's run headless. Not absolutely sure why as I didn't investigate it further but headless Chrome is known for sometimes causing different behaviors